### ISSUE TITLE
Add --no-upgrade (or equivalent) flag to `brew bundle check`

### ISSUE THREAD

**Issue by jhottenstein** [Homebrew/homebrew-bundle#1158]
I would like to use a Brewfile in my project directory to handle system dependencies and then add something like
```
  system('brew bundle check') || system!('brew bundle --no-upgrade --no-lock')
```
to my bin/setup file.

I would like to tell `brew bundle check` to return 0 if the dependencies are installed but not care if they are upgraded.  

Happy to contribute a PR if this behavior is desired.

Thanks!

**Comment by MikeMcQuaid** [comment]
Could you just remove the `system('brew bundle check')` here instead?

**Comment by jhottenstein** [comment]
Yup!  That's what i've done for now.

But I just dug into the code and I think this behavior is already implemented.  Sorry for the spurious report.

I ran into something weird with the mysql formula where it kept renotifying me that mysqld_safe was allowed to run in the background.  It also prevented the `brew bundle check --no-upgrade` from completing successfully.

I'll reopen if I noticed this issue again.

Thanks for the quick response!
